---
layout: post
title: Writing something 
subtitle: Learn to run a gwas
gh-repo: https://github.com/rprendes
cover-img: /assets/img/Image1.png (https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5889-5/figures/2)
gh-badge: [star, fork, follow]
tags: [gwas,plink,genotypes]
comments: true
---

I wanted to create a very basic guide to perfom a GWAS for people without knowledge about bioinformatics (like me). In this [script]  (https://github.com/rprendes/common-use-scripts/blob/main/script.GWAS_MixedModel.sh) you can find the steps.

It covers from imputs from plink to plot the results using the linear mixed model implemented in this [software] https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3386377/ in unix and qqman library in R. 


**...**
